CMSgov/easi-app
# Easi 1473/bookmark system card

# EASI-1473

## Changes proposed in this pull request

- Added UI components for Bookmarking systems list items.  This is UI only, and not integration with backend db.

## Reviewer Notes

- Bookmarks are in the sandbox view as a part of the larger systems list story.
- Not all elements on this Figma board were required for story (pagination, page headers, etc)
- Figma - [https://www.figma.com/file/RhVQgXxE6tyj33TygymuhD/System-List-and-Profile-(Hi-fi)?node-id=77%3A154](url)
- Mocked CEDAR/Bookmark db data used
- Clicking bookmark icon functionality from table is not a part of this story
- All bookmarks and text should be mobile/tablet responsive
- Truss/USWDS components were utilized for for CardGroup and Card components
- Fields, headers, icon-status relations, etc. don't seem to be solidified yet (dependent on CEDAR/db responses), but are variables of the card itself.

## Setup & How to test

- Pull the branch locally and test it
- Check for mobile responsiveness
- Check that all code is adequately covered by tests
- Make comments, even if it's minor!



## PR Author Review Checklist

functions, or e2e tests as necessary
the [rotor menu](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)
- For any new migrations/schema changes:

## Screenshots
